docs/asciidoc/modules/ROOT/pages/background-operations/triggers.adoc

```diff
@@ -365,27 +393,58 @@ because using this one, the rebuild of entities is executed automatically under
 .Other examples
 [source,cypher]
 ----
-CALL apoc.trigger.add('timestamp','UNWIND $createdNodes AS n SET n.ts = timestamp()', {});
-CALL apoc.trigger.add('lowercase','UNWIND $createdNodes AS n SET n.id = toLower(n.name)', {});
-CALL apoc.trigger.add('txInfo',   'UNWIND $createdNodes AS n SET n.txId = $transactionId, n.txTime = $commitTime', {phase:'after'});
-CALL apoc.trigger.add('count-removed-rels','MATCH (c:Counter) SET c.count = c.count + size([r IN $deletedRelationships WHERE type(r) = "X"])', {})
-CALL apoc.trigger.add('lowercase-by-label','UNWIND apoc.trigger.nodesByLabel($assignedLabels,"Person") AS n SET n.id = toLower(n.name)', {})
+CALL apoc.trigger.install('dbTest', 'timestamp','UNWIND $createdNodes AS n SET n.ts = timestamp()', {});
+CALL apoc.trigger.install('dbTest', 'lowercase','UNWIND $createdNodes AS n SET n.id = toLower(n.name)', {});
+CALL apoc.trigger.install('dbTest', 'txInfo',   'UNWIND $createdNodes AS n SET n.txId = $transactionId, n.txTime = $commitTime', {phase:'after'});
+CALL apoc.trigger.install('dbTest', 'count-removed-rels','MATCH (c:Counter) SET c.count = c.count + size([r IN $deletedRelationships WHERE type(r) = "X"])', {})
+CALL apoc.trigger.install('dbTest', 'lowercase-by-label','UNWIND apoc.trigger.nodesByLabel($assignedLabels,"Person") AS n SET n.id = toLower(n.name)', {})
 ----
 
 // end::trigger[]
 
-.Trigger Phase Table
-.Helper Functions
-[cols="1m,5"]
+
+== Remove triggers
+
+If we want to remove the trigger with name 'test' in 'neo4j' database, 
+we can run the following query:
+
+[source,cypher]
+----
+CALL apoc.trigger.drop('neo4j', 'test')
+----
+
+If we want to remove all triggers in 'neo4j' db, we can execute
+
+[source,cypher]
+----
+CALL apoc.trigger.dropAll('neo4j')
+----
+
+
+== List of triggers
+
+If we want to get the list of triggers, for example if we have one trigger created via:
+
+[source,cypher]
+----
+CALL apoc.trigger.install('neo4j', 'count-removals', 
+    'MATCH (c:Counter) SET c.count = c.count + size([f IN $deletedNodes WHERE id(f) > 0])', 
+    {})
+----
+
+we can execute (also in this case, after a time defined by the configuration `apoc.trigger.refresh`):
+
+[source,cypher]
+----
+CALL apoc.trigger.list
```

Comment:
Should this be changed to apoc.trigger.show() or do you want to do that later? I don't mind either way, this is just a reminder.

Reply:
Personally I would do it in the other pr :)